launch.go

```diff
@@ -143,13 +145,17 @@ func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 	AccountServiceLogOutURL := getAccountServiceURL(r)
 	urlValues := r.URL.Query()
 	surveyURL := urlValues.Get("url")
+	defaultValues := authentication.GetDefaultValues()
 	log.Println("Quick launch request received", surveyURL)
 
-	urlValues.Add("ru_ref", authentication.GetDefaultValues()["ru_ref"])
-	urlValues.Add("collection_exercise_sid", uuid.NewV4().String())
-	urlValues.Add("case_id", uuid.NewV4().String())
+	urlValues.Add("ru_ref", defaultValues["ru_ref"])
+	collectionExerciseSid, _ := uuid.NewV4()
+	caseID, _ := uuid.NewV4()
```

Comment:
Seems to be a mix of cases on vars with 'ID'?

Reply:
Yeah, I think this might be an exception specific to go? My editor auto linted it from `caseId` to `caseID` (using golint). In the case above, it is an `Sid` which doesn't have an exception I guess.